### PR TITLE
[UR] Modify fill emulation to work for patterns which are not powers of 2

### PR DESCRIPTION
**Pull request by konradkusiak97**
https://github.com/oneapi-src/unified-runtime/pull/1603

### CODE REVIEW

**Comment by konradkusiak97** [comment]
failures on gen12 not related

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -109,8 +109,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
   )
 
   fetch_adapter_source(opencl
-    ${UNIFIED_RUNTIME_REPO}
-    ${UNIFIED_RUNTIME_TAG}
+    "https://github.com/konradkusiak97/unified-runtime.git"
+    queueFillOPENCLctsFix
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:113]:
```suggestion
    ${UNIFIED_RUNTIME_REPO}
    # commit e16d01c02f1c1c18af19d3c882c08f791c2345d9
    # Merge: a3895dbf 483a6325
    # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
    # Date:   Thu May 30 10:04:43 2024 +0100
    #     Merge pull request #1603 from konradkusiak97/queueFillOPENCLctsFix
    #     [OpenCL] Modify fill emulation to work for patterns which are not powers of 2
    e16d01c02f1c1c18af19d3c882c08f791c2345d9
```

**Review by kbenzie — COMMENTED** [review]
https://github.com/oneapi-src/unified-runtime/pull/1603 has been merged, please pull in the latest sycl branch changes and update OpenCL UR tag as suggested.

**Review by kbenzie — APPROVED** [review]

**Comment by kbenzie** [comment]
@intel/llvm-gatekeepers please merge

**Pull request merged**